demo/transient/transient_snes_mpfaof90.F90

```diff
@@ -184,6 +185,10 @@ program main
   CHKERRA(ierr);
   call PetscOptionsGetBool(PETSC_NULL_OPTIONS,PETSC_NULL_CHARACTER,"-pflotran_consistent",pflotran_consistent,flg,ierr);
   CHKERRA(ierr)
+
```

Comment:
Is this a temporary approach for switching this demo to use the driver? Are we planning to start using driver logic exclusively at some point in the future?

Reply:
>Are we planning to start using driver logic exclusively at some point in the future?

You read my mind. I envision switching all 3D+MPFA-O demos to use the driver instead of directly calling `SNESSolve` or `TSSolve`. A new PR will be coming soon.